### DISCUSSION TITLE
glossary: A glossary for outbreak analytics terms

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/epiverse-trace.github.io#100] [Project Ideas]
### Description

This repository will register brief definitions of concepts in the field. This repository will also register all the working definitions from discussions in different repositories, issues, and discussions.

Here are three examples:
- https://github.com/epiverse-trace/bpmodels/discussions/63
- https://github.com/epiverse-trace/epiparameter/issues/115
- https://github.com/orgs/epiverse-trace/discussions/81

This idea was supported by @Degoot-AM and @CarmenTamayo during discussions on casestudies. Please, take a look at the **Related projects** section. I would like to read your thoughts on what format to choose for this.

### Typical end-users

I think in people like [Juan](https://epiverse-trace.github.io/personas/juan-datero.html), who is novice in Epidemiological concepts; [Lucia](https://epiverse-trace.github.io/personas/lucia-outbreaks.html), who is novice in IDD concepts; and [Danielle](https://epiverse-trace.github.io/personas/danielle-entretrainer.html), who is interested to share precise definitions for concepts in the field.

### Potential contributors

People like [Patricia](https://epiverse-trace.github.io/personas/patricia-discoverer.html) and [Vania](https://epiverse-trace.github.io/personas/vania-academica.html) who have working expirience in the field at Universities. Also target users comfortable contribuing to GitHub projects like Juan and Danielle.

### Key collaborators

No

### Inputs

Not apply

### Outputs

Not apply

### Imports

Not apply

### Used by

Not apply

### Related projects

* https://glosario.carpentries.org/
* https://debruine.github.io/glossary/
* https://hbiostat.org/glossary/

### Usage

```bash
Not apply
```

### Additional comments

Users will:
- reuse links from available definitions to redirect readers to them for referenced definitions.
- submit definitions while working on a new analysis project or R package.

**Comment by avallecam** [comment]
@amanda-minter do you think that with a glossary embedded in the tutorials ([link](https://github.com/epiverse-trace/tutorials/issues/32)) we are already solving this project idea? Thus, it would not be required to create a new glossary repository.

**Comment by amanda-minter** [comment]
Yes I think so, and it would be a useful link to the tutorials. The glossary is hosted [here](https://epiverse-trace.github.io/tutorials/reference.html). 

The glossary is written in [pandoc format ](https://github.com/epiverse-trace/tutorials/blob/main/learners/reference.md), tags can be added to create links to particular entries e.g. https://epiverse-trace.github.io/tutorials/reference.html#indirect

**Comment by avallecam** [comment]
Thank you, Amanda. I'll close this and link a new issue to move the examples on top to the corresponding glossary https://github.com/epiverse-trace/tutorials/issues/65
